### PR TITLE
ref(ui): remove inline settings icon from project dropdown

### PR DESCRIPTION
**Pull request by ckj**

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

**Review by lynnagara — APPROVED** [review]

**Review by dcramer — APPROVED** [review]
looks like code, removes stuff per title

**Review by lynnagara — APPROVED** [review]

**Review by benvinegar — APPROVED** [review]

**Review by mattrobenolt — APPROVED** [review]

**Review by markstory — APPROVED** [review]
:clap:

**Review by wedamija — APPROVED** [review]

**Review by JTCunning — APPROVED** [review]
🍪

**Comment by thekashifmalik** [comment]
lgtm

**Review by alex-hofsteede — APPROVED** [review]

**Comment by alex-hofsteede** [comment]
We're gonna look like a bunch of idiots if this actually breaks something.

**Review by connorshea — APPROVED** [review]

**Review by logicalbomb — APPROVED** [review]
:shipit: Always approve code deletion!

**Review by mitsuhiko — APPROVED** [review]
lgtm

**Review by untitaker — APPROVED** [review]
ok

File: src/sentry/static/sentry/app/components/projectSelector.jsx [hunk 1]
```diff
@@ -303,19 +293,6 @@ const BookmarkIcon = styled(InlineSvg)`
...
-  padding: ${space(0.5)};
-  opacity: 0;
-  transform: translateX(-${space(0.5)});
-  transition: 0.2s all;
-
-  &:hover {
-    color: ${p => p.theme.gray4};
-  }
-`;
-
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/projectSelector.jsx]: I like the way you delete these lines of code. thats good

**Review by evanpurkhiser — APPROVED** [review]

**Review by kamilogorek — APPROVED** [review]
💯

**Comment by mitsuhiko** [comment]
Well done

**Comment by gddh** [comment]
lgtm 2

**Comment by wedamija** [comment]
Congrats

**Pull request merged**